core/src/main/scala/kafka/cluster/Partition.scala

```diff
@@ -497,7 +498,7 @@ class Partition(val topicPartition: TopicPartition,
 
       val leaderLog = localLogOrException
       val leaderEpochStartOffset = leaderLog.logEndOffset
-      info(s"$topicPartition starts at leader epoch ${partitionState.leaderEpoch} from " +
+      stateChangeLogger.info(s"Leader $topicPartition starts at leader epoch ${partitionState.leaderEpoch} from " +
```

Comment:
I think this made it clearer and easy to separate from the `makeFollower()` log, even though both have a log line indicating that we're handling a `become-leader` or `become-follower` request